mtags/src/main/scala-2.12/scala/jdk/CollectionConverters.scala

```diff
@@ -0,0 +1,5 @@
+package scala.jdk
```

Comment:
Let's define this in our own `scala.meta.internal.jdk` package to prevent conflicts with scala-compat

Reply:
Context: mtags is published as a standalone library. I don't think anybody is using it as a library besides Metals but we should nevertheless try to be good citizen

Reply:
makes sense 👍